plugins/source/github/resources/plugin/plugin.go

```diff
@@ -0,0 +1,30 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
+)
+
+const (
+	exampleConfig = `
+		access_token: <Access Token HERE>
+		orgs:
+		- cloudquery
+`
```

Comment:
```suggestion
	exampleConfig = `
access_token: <Access Token HERE>
orgs:
- cloudquery
`
```

I believe this should not be indented

Reply:
When we use indentation then we get the properly constructed example config.
Alternatively, we should go to filling in the Spec with the example data and marshal it to YAML altogether.

Reply:
@candiduslynx I'm not sure how you checked, but using this branch, I just ran the plugin server with `go run main.go serve` and then use the cli to run `go run main.go generate source --registry grpc 127.0.0.1:7777`. The generated config was not correctly indented:

![Screenshot 2022-09-12 at 08 55 36](https://user-images.githubusercontent.com/1121616/189601785-c941741a-d2cd-42ea-a32b-b5adf1fb1f21.png)

The code for the `generate` command adds the necessary indentation to get it at the right indentation level, we don't need to do it here. 

Personally I think an example string is fine, but if you want to change it to struct marshalling to yaml, that's great too, let's just make sure there's no indent either way.